emails/views.py

```diff
@@ -556,9 +566,19 @@ def _handle_bounce(message_json):
 
 
 def _get_text_html_attachments(message_json):
-    bytes_email_message = message_from_bytes(
-        message_json['content'].encode('utf-8'), policy=policy.default
-    )
+    if 'content' in message_json:
+        message_content = message_json['content'].encode('utf-8')
+    elif 'receipt' in message_json and 'action' in message_json['receipt']:
+        if 'S3' in message_json['receipt']['action']['type']: 
+            bucket = message_json['receipt']['action']['bucketName']
+            object_key = message_json['receipt']['action']['objectKey']
+            message_content = get_message_content_from_s3(bucket, object_key)
+            histogram_if_enabled(
+                'relayed_email.size',
+                len(message_content)
+            )
```

Comment:
@groovecoder, using a histogram here so we can look at the data as an individual and in aggregate. Let me know if you think the name should be changed or anything?

Reply:
Nice!